src/mlagility/analysis/util.py

```diff
@@ -80,6 +82,53 @@ def count_parameters(model: torch.nn.Module, model_type: build.ModelType) -> int
     raise AnalysisException(f"model_type {model_type} is not supported")
 
 
+def get_onnx_ops_list(onnx_model) -> Dict:
+    """
+    List unique ops found in the onnx model
+    """
+    onnx_ops_counter = defaultdict(int)
+    try:
+        model = onnx.load(onnx_model)
+    except Exception as e: # pylint: disable=broad-except
+        print (f"Failed to get ONNX ops list from {onnx_model}: {str(e)}")
```

Comment:
We have been using `groqflow.common.printing.log_warning` for printing warnings like those.